README.md

```diff
@@ -1 +1,7 @@
 # hw01
+## Surrein Maniraju Homework 1
+Hi my name is *Surrein Maniraju*. I am a fourth year Sociology Major in the college. I am taking this class because there are many things I am interested in that require computing skills such as:
+
+* GIS
+* Network Analysis
+* Data Scraping from sites such as [Twitter](https://twitter.com/?lang=en)
```

Comment:
Because most of us are familiar with Twitter, I don't think you necessarily needed to make this a link. I would have loved to be connected to a webpage that gave me more information about something you were interested in.